test/xcommon_cmake/src/main.c

```diff
@@ -0,0 +1,32 @@
+
```

Comment:
Since you have printf calls in this source file, it should explicitly include stdio.h rather than relying on something else to include it.